specs/cm/change-mgt-shapes.html

```diff
@@ -2,30 +2,29 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>OSLC Change Management Version 3.0. Part 3: Constraints</title>
+    <title>OSLC Change Manazgement Version 3.0. Part 3: Constraints</title>
 
     <meta
       name="description"
       content="Specifies the standard constraints on the OSLC Change Management domain vocabulary terms."
     />
 
     <script
-      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.17/builds/respec-oasis-common.min.js"
+      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.20/builds/respec-oasis-common.min.js"
       async
       class="remove"
     ></script>
-    <!-- <script src='http://127.0.0.1:9000/respec-oasis-common.js' async class='remove'></script> -->
```

Comment:
This is actually needed to test ReSpec from the master branch... Why did you remove it if I may ask?

Reply:
I thought they were old an no longer needed.